pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "search"
+version = "0.1.0"
+description = "Libero search service"
+authors = ["Giancarlo Fusiello"]
```

Comment:
I'd prefer not including names in repos.

Reply:
name removed.